pkgs/applications/version-management/debase/default.nix

```diff
@@ -0,0 +1,90 @@
+{
+  darwin,
+  fetchFromGitHub,
+  fetchpatch, # Delete at next version bump.
+  lib,
+  libgit2,
+  stdenv,
+}:
+stdenv.mkDerivation rec {
+  pname = "debase";
+  version = "2";
+
+  src =
+    (fetchFromGitHub {
+      owner = "toasterllc";
+      repo = "debase";
+      rev = "refs/tags/v${version}";
+      hash = "sha256-FIWGJtjmLpvxHxDbmbL55FKNkRaBvcPsdWimGWdPfmU=";
+      fetchSubmodules = true;
+      leaveDotGit = true;
```

Comment:
Can we avoid this? `leaveDotGit` can be unreproducible. If there are just some commands to get version, hash, etc we can substitute them.

Reply:
I didn't know that. Thanks. We only needed it for the commit hash, so I instead I'm just including the hash as a literal string: https://github.com/NixOS/nixpkgs/pull/318702/files#diff-319911f9c744f0ad71f957f7d339493a2843a178e11cf7a45f18e4af5f57bb0aR37

I wanted to get the hash programmatically, but I don't know a way to do it in the sandbox without `leaveDotGit`. I suppose it's easy enough to update it manually. I added comments reminding future editors to do so.